sample/mpp-library/nested-module/build.gradle.kts

```diff
@@ -15,6 +12,7 @@ kotlin {
     android()
     ios()
     macosX64()
+    jvm()
```

Comment:
we not need all this targets declaration - we setup targets in convention plugin

Reply:
fixed